tests/collections/ttables.nim

```diff
@@ -80,7 +80,7 @@ block thashes:
   # Test with range
   block:
     type
-      R = range[1..10]
+      R = range[0..9]
```

Comment:
why does this test need to change?

Reply:
Because it produces a new warning, I think.

Reply:
no this PR actually suppresses the warning instead (which is great); so IMO the test should keep `[1..10]`, and the 2 `# causes warning, why?` can finally disappear. yay.